tests/functional/search-auth.js

```diff
@@ -90,7 +91,29 @@ define([
           assert.include(currUrl, 'q=vladvlad', 'Our params didn\'t go anywhere.');
         })
         .end();
-    }
+    },
 
+    'Search from the homepage': function() {
+      return this.remote
+        .setFindTimeout(intern.config.wc.pageLoadTimeout)
+        .get(require.toUrl(url('/')))
+        .findByCssSelector('.js-SearchBarOpen').click()
+        .end()
+        .findByCssSelector('.js-SearchBar input').click()
```

Comment:
Hmm, seems like Selenium does it too quickly. I guess we just leave it in. :fried_shrimp: